Controllers/loginController.js

```diff
@@ -0,0 +1,31 @@
+const fs = require("fs");
+
+const login = async (req, res) => {
+  try {
+    let userData = JSON.parse(fs.readFileSync("./data/user.json"));
+
+    if (
+      userData.username == req.body.username &&
+      userData.password == req.body.password
```

Comment:
```suggestion
      userData.username === req.body.username &&
      userData.password === req.body.password
```
Let's try to use the strict equality as much as possible. It will avoid any unforeseen outcomes. Using loose equality here means that it will only check the value and not the type (for ex. "123" and 123 both will return true)

Reply:
change it.
555030a5724922608f6b646c8dd7e1021724588a